src/containers/Tenant/Schema/SchemaTree/SchemaTree.tsx

```diff
@@ -50,33 +59,80 @@ export function SchemaTree(props: SchemaTreeProps) {
         return childItems;
     };
 
+    React.useEffect(() => {
+        if (createDirectoryResult.isSuccess) {
+            setNavigationTreeKey(createDirectoryResult.requestId);
+            onActivePathUpdate(`${parent}/${child}`);
+            setCreateDirectoryOpen(false);
+        }
+    }, [createDirectoryResult, onActivePathUpdate, parent, child]);
+
+    React.useEffect(() => {
+        if (createDirectoryResult.isError) {
+            const errorMessage =
+                (createDirectoryResult.error as {data: string})?.data || ' Unknown error';
```

Comment:
Does the error contain a `message`?

Reply:
The error contains only such fields
`{
    "data": "{ status: SCHEME_ERROR, issues: { <main>: Error: Check failed: path: '/local/.sys_health/±', error: symbol '?' is not allowed in the path part '±' } }",
    "status": 400,
    "statusText": "Bad Request",
    "headers": {
        "content-length": "140",
        "content-type": "text/plain"
    },
    "config": {
        "transitional": {
            "silentJSONParsing": true,
            "forcedJSONParsing": true,
            "clarifyTimeoutError": false
        },
        "adapter": [
            "xhr",
            "http"
        ],
        "transformRequest": [
            null
        ],
        "transformResponse": [
            null
        ],
        "timeout": 60000,
        "xsrfCookieName": "",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1,
        "env": {},
        "headers": {
            "Accept": "application/json, text/plain, */*",
            "Content-Type": "application/json"
        },
        "withCredentials": false,
        "signal": {},
        "method": "post",
        "url": "http://localhost:8765/scheme/directory",
        "data": "{}",
        "params": {
            "database": "/local/.sys_health",
            "path": "/local/.sys_health/±"
        },
        "axios-retry": {
            "retries": 3,
            "shouldResetTimeout": false,
            "validateResponse": null,
            "retryCount": 0,
            "lastRequestTime": 1719906437941
        }
    },
    "request": {}
}`